components/Pagination/style.module.scss

```diff
@@ -0,0 +1,73 @@
+@import "styles/index";
+
+.pagination {
+  border-top: 1px solid $color-night;
+  border-bottom: 1px solid $color-night;
+  padding: 6px 0;
+  position: relative;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__info {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  &__info-viewing-label {
+    text-transform: uppercase;
+    font-weight: 700;
+  }
+
+  &__nav {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__nav-item {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--active > * {
+      text-shadow: 0 0 1px $color-night;
+    }
+  }
+
+  &__nav-item-prev {
+    margin-right: 10px;
```

Comment:
Possibly snap to spacing size? If it looks weird and this is justified, I totally understand. I just wanted to mention them.

Reply:
@tarikhamilton Updated to use `$space-xs`. Still looks good IMO.